test/qta/case_study.qta

```diff
@@ -69,6 +69,7 @@ start default
   end application
 
   define application "Chiller"
+    # Chillers are large size ACs that other simulations may put under the commercial AC category but it should be its own application category for the case study
```

Comment:
Please break this into multiple lines with each having a comment and each line be no more than 100 characters long